gax-httpjson/version.txt

```diff
@@ -0,0 +1 @@
+0.22.1-SNAPSHOT
```

Comment:
should this have its own version from 0.0.1? I don't think it's a big deal either way, but it feels funny that http and grpc are version-locked.